content/modules/r/pages/data-frames.adoc

```diff
@@ -0,0 +1,348 @@
+= data.frames
+
+{sp} +
+
+== Abstract
+
+Data.frames are one of the primary data structures used very frequently when working in R. Data.frames are tabular data structures, consisting of named columns of equal size, where each column has a single type.
+
+You can create a data.frame easily. Vectors are used to fill the columns as follows:
```

Comment:
Creating a data.frame is easily done by filling in the columns using vectors.  (this would be a good place to explain what you are doing in the code)